compiler/plugins/input/StableHLO/Conversion/Passes.cpp

```diff
@@ -40,6 +41,12 @@ void registerStableHLOConversionPassPipeline() {
 // Prepare HLO for use as an input to the Flow dialect.
 void buildStableHLOInputConversionPassPipelineImpl(
     OpPassManager &passManager, const StableHloOptions &options, bool detuple) {
+  passManager.addPass(stablehlo::createCheckVHLOStableHloMixUsage());
+  // If the input is VHLO, then it is automatically converted to Stablehlo
+  // otherwise, this pass is considered a NOP.
+  // Note: Having StableHLO and VHLO in the same function is considered
+  // undefined behavior.
+  ::mlir::stablehlo::createStablehloDeserializePipeline(passManager);
```

Comment:
At the bottom of this function, could you also add vhlo as an illegal dialect in `stablehlo::createVerifyCompilerStableHloInputLegality`?